### PR TITLE
latin spanish content

### PR DESCRIPTION
**Pull request by diegoaguilar**
Done!

### CODE REVIEW

File: content/es.kindly [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The KINDLY License
+Copyright (c) <year> <author>
+
+Tienes la autorización para utilizar este software, leer su código, modificarlo y distribuirlo bajo las siguientes condiciones:
```
Comment (IonicaBizau) [inline content/es.kindly]: Thanks! :sparkler:

Please format it to have max 80 columns, like in `license.kindly`. Also, please move the `content.kindly` file into `content/en.kindly`.

**Comment by IonicaBizau** [comment]
@diegoaguilar Awesome! I will modify the build scripts so they will handle i18n. BTW, add yourself as contributor (see [point 5](https://github.com/IonicaBizau/kindly-license/blob/gh-pages/CONTRIBUTING.md)).

**Comment by diegoaguilar** [comment]
@IonicaBizau I've added myself as contributor

**Comment by IonicaBizau** [comment]
Better later than never. :smile: Sorry for taking so long.

:ship: `1.2.0` :tada: 

Thanks!!! :apple:

**Pull request merged**